ios/PolyPodApp/PolyPodUITests/OnboardingTest.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+
+class OnboardingTest: XCTestCase {
+    let app = XCUIApplication()
+    
+    override func setUp() {
+        continueAfterFailure = false
+    }
+    
+    func testOnboardingShowsUp() {
+        launchApp(firstRun: true)
+        // TODO: Assert onboarding screen is shown
+        XCTAssertTrue(true)
```

Comment:
? This to be removed later?

Reply:
Of course, this is not ready to be reviewed - just some WIP changes